modules/artifacts_bucket.tf

```
@@ -1,6 +1,6 @@
 locals {
   principal_policy     = var.principal_policy == "" ? "${path.module}/fixtures/policies/principal_policy.tmpl" : var.principal_policy
-  artifact_bucket_name = "${local.account_id}-dce-artifacts-${var.namespace}"
+  artifact_bucket_name = "dce-artifacts-${var.namespace}"
```

```suggestion
  artifact_bucket_name = "${local.account_id}-dce-artifacts-${var.namespace}"
```
S3 buckets need to be globally unique. Also, changing this bucket name will be a breaking change for those using the module.

---

Hit a random issue that night that stopped me creating a bucket with the AccountID at the start. Working now so this change has been reverted.